src/game_objects/essence.h

```diff
@@ -29,6 +32,7 @@ class Essence : public GameObject {
     std::unique_ptr<collectibles::Collectible> value;
 
     std::function<void()> cancel_fade;
+    std::function<void()> cancel_timeout;
     std::function<void()> cancel_rainbow; // Cancels the constant rainbow color changing
```

Comment:
Initialize these to no-ops (`[](){}`) so you don't have to check before calling.